library/src/common/utils.ts

```diff
@@ -294,6 +300,116 @@ const getQueryParams = <Q>(query: Q): string => {
   return '';
 };
 
+const appInfoToUserAgentString = (appInfo: ApplicationInfo): string =>
+  `(${appInfo.name ? appInfo.name : ''}; ${
+    appInfo.version ? appInfo.version : ''
+  }; ${appInfo.url ? appInfo.url : ''})`;
+
+const buildUserAgentString = (appInfo?: ApplicationInfo): string => {
+  let userAgent = `${USER_AGENT_CLIENT}/${process.env.VERSION ?? 'unknown'}`;
```

Comment:
if `VERSION === ''`, this will result in `${USER_AGENT_CLIENT}/`;

The following will result in `unknown`
```suggestion
  let userAgent = `${USER_AGENT_CLIENT}/${process.env.VERSION || 'unknown'}`;
```